submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/files/service/GroupFilesService.kt

```diff
@@ -41,7 +41,7 @@ class GroupFilesService {
             .resolve(path)
             .resolve(fileName)
             .toFile()
-        FileUtils.deleteFile(file)
+        FileUtils.deleteIfExists(file)
```

Comment:
I think we should not do make the api ignore error. (someone trying to delete a file that does not exists).  Is ok and IMO expected that api fail if someone request to delete a file that does not exits.